src/routes/profile.js

```diff
@@ -155,30 +193,139 @@ router.get(
   },
 );
 
+const storage = multer.memoryStorage();
+const upload = multer({ storage: storage }).fields([
+  { name: 'avatarRes', maxCount: 1 },
+  { name: 'avatarSpy', maxCount: 1 },
```

Comment:
Can you also add a comment saying this is a whitelist and other files won't be accepted by the server.